.github/issue_template.md

```diff
@@ -0,0 +1,27 @@
+### If it's a discussion, please start one [here](https://github.com/cultureamp/kaizen-design-system/discussions).
+
+- [ ] Before opening this issue, I've searched open and closed issues for a fix.
+
+## Expected Behavior
+
+
+## Actual Behavior (please provide screenshots)
+
+
+## Steps to Reproduce the Problem
+
+1.
+2.
+3.
+
+## Specifications (please add any other notable libs/details)
+
+- Kaizen version:
+- Repo:
```

Comment:
When I read this, my first though was "won't the repo always be kaizen-design-system?"

But I'm guessing this is about a bug where the problem might have been encountered? Is there a way to capture that in the name?

In our team we often talk about "consuming repos" but I am not sure that language would be obvious to people outside our team

Reply:
I was more going for "what repo you're working in?" for our context. Like `survey-summary-frontend` or maybe that's not useful to us?

Reply:
If it's an issue in production, the repo and/or location in the product is useful. If it's an issue someone has encountered in work that's in progress, a branch is useful. It would be hard to capture these things succinctly - I wouldn't be opposed to excluding this for brevity and let whoever opened the issue to provide whatever information they think is useful.